coulomb/js/src/main/scala/coulomb/typename.scala

```diff
@@ -0,0 +1,54 @@
+package coulomb
+
+import scala.language.experimental.macros
+
+import scala.reflect.macros.whitebox
```

Comment:
are both blackbox and whitebox used here?

Reply:
With my basic understanding of macros I think whitebox are preferred but I could try if the test work in either case
I think only one should be used

Reply:
As I understand it, any particular macro is one or the other.  So I'm guessing only one of whitebox or blackbox needs to be imported.

Reply:
Ok, I'll clean this up a bit, I also thought I could move the `=:=` operation to another typeclass that is only used in testing

Reply:
I don't need `blackbox` at all